Building from Dockerfile causes segmentation fault

**Describe the bug**
When building Clouditor from branch `main` as a container image using the provided `Dockerfile` causes a segmentation fault during `go generate ./...`. The following error is printed:
```
...
go: downloading github.com/xeipuuv/gojsonpointer v0.0.0-20190905194746-02993c407bfb
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x898f4a]

goroutine 1 [running]:
github.com/googleapis/gnostic/apps/protoc-gen-openapi/generator.(*OpenAPIv3Generator).addPathsToDocumentV3(0x2, 0xc000023680, 0xc0002f7b00)
	/go/pkg/mod/github.com/googleapis/gnostic@v0.5.5/apps/protoc-gen-openapi/generator/openapi-v3.go:136 +0x26a
github.com/googleapis/gnostic/apps/protoc-gen-openapi/generator.(*OpenAPIv3Generator).buildDocumentV3(0xc00029aeb0)
	/go/pkg/mod/github.com/googleapis/gnostic@v0.5.5/apps/protoc-gen-openapi/generator/openapi-v3.go:83 +0x170
github.com/googleapis/gnostic/apps/protoc-gen-openapi/generator.(*OpenAPIv3Generator).Run(0xc00029aeb0)
	/go/pkg/mod/github.com/googleapis/gnostic@v0.5.5/apps/protoc-gen-openapi/generator/openapi-v3.go:57 +0x25
main.main.func1(0x0)
	/go/pkg/mod/github.com/googleapis/gnostic@v0.5.5/apps/protoc-gen-openapi/main.go:25 +0x1e
google.golang.org/protobuf/compiler/protogen.run({0x0, 0x0}, 0x9a35d8)
	/go/pkg/mod/google.golang.org/protobuf@v1.27.1/compiler/protogen/protogen.go:74 +0x142
google.golang.org/protobuf/compiler/protogen.Options.Run({0x0, 0x0}, 0xc0000001a0)
	/go/pkg/mod/google.golang.org/protobuf@v1.27.1/compiler/protogen/protogen.go:52 +0x2a
main.main()
	/go/pkg/mod/github.com/googleapis/gnostic@v0.5.5/apps/protoc-gen-openapi/main.go:24 +0x25
--openapi_out: protoc-gen-openapi: Plugin failed with status code 2.
proto.go:32: running "protoc": exit status 1
```

**To Reproduce**
Steps to reproduce the behavior:
1. Build Clouditor container image from `Dockerfile`: `$ docker build -t clouditor .`
2. Error occurs.

**Expected behavior**
Docker build should run without errors and produce container image.

**Additional context**
I'm running on latest Debian testing/bookworm with Docker version 20.10.13.

--- a/Dockerfile
+++ b/Dockerfile
@@ -15,7 +15,8 @@ RUN go install google.golang.org/protobuf/cmd/protoc-gen-go \
 ADD . .
 
 RUN go generate ./...
-RUN go build ./...
+RUN go build -o /build/engine cmd/engine/engine.go
+RUN go build -o /build/cl cmd/cli/cl.go
 
 FROM debian:stable-slim
 

--- a/Dockerfile
+++ b/Dockerfile
@@ -10,7 +10,7 @@ RUN apt update && apt install -y protobuf-compiler
 RUN go install google.golang.org/protobuf/cmd/protoc-gen-go \
     google.golang.org/grpc/cmd/protoc-gen-go-grpc \
     github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway \
-    github.com/googleapis/gnostic/apps/protoc-gen-openapi
+    github.com/google/gnostic/cmd/protoc-gen-openapi
 
 ADD . .
 

--- a/README.md
+++ b/README.md
@@ -43,7 +43,7 @@ Also make sure that `$HOME/go/bin` is on your `$PATH` and build:
 
 ```
 go generate ./...
-go build ./...
+go build -o ./engine cmd/engine/engine.go
 ```
 
 ## Usage

